app.ts

```diff
@@ -0,0 +1,312 @@
+        <label class="record-labels" for="record-id"
+          >Enter record ID :
+        </label>
+        <input
+          type="text"
+          name="record-id"
+          id="record-id"
+          class="navigation-input"
+          value="0"
+        />
+      </div>
+      <p class="amount-of-records"></p>
+    </div>
+  <button id="get-record-btn">See Record</button>
+  `;
+
+        recordNav.innerHTML = singleRecordSelection;
+
+        let returnBtn: HTMLElement | null = document.getElementById("return-btn");
+        let recordIdInput: HTMLElement | null = document.getElementById("record-id");
+        let numberOfRows = Math.floor(window.innerHeight / 50);
+        let getSingleRecord: HTMLElement | null = document.getElementById("get-record-btn");
+
+        returnBtn?.addEventListener("click", () => {
+            console.log("hello");
+            createNavigation();
+            fromNumber = 0;
+            let toNumber = fromNumber + numberOfRows;
+            getRecords(fromNumber, toNumber);
+        });
+
+        getSingleRecord?.addEventListener("click", () => {
+            let recordIdValue = (recordIdInput as HTMLInputElement).value;
+            fromNumber = Number(recordIdValue);
+
+            console.log(fromNumber);
+
+            let toNumber = fromNumber + numberOfRows;
+            let finalRecord = recordNumberTotal - 1;
+
+            if (toNumber > finalRecord) {
+                toNumber = finalRecord;
+                fromNumber = toNumber - numberOfRows;
+            }
+
+            let check = ["undefined", "string", ""];
+
+            if (check.includes(typeof fromNumber) || fromNumber < 0) {
+                alert("Does not exists");
+                recordIdValue = "0";
+            } else if (typeof fromNumber === "number" && fromNumber >= 0) {
+                getRecords(fromNumber, toNumber);
+            }
+        });
+    }
+}
+
+function createHeadingGrid(headings: any) {
+    let headingColumns: HTMLElement | null = document.getElementById("column-headings-container"); // Headings
+    let headingsData = `<h1 class="column-heading">${headings}</h1>`;
+
+    (headingColumns as HTMLDivElement).innerHTML += headingsData;
+}
+
+function recordCount() {
+    return fetch("http://localhost:2050/recordCount", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(checkResponseError)
+        .then((response: Response) => response.text())
+        .then((data) => {
+            recordNumberTotal = JSON.parse(data);
+        })
+        .catch((error: Error) => {
+            console.log(error);
+        });
+}
+
+function headingRowCreation() {
+    return fetch("http://localhost:2050/columns", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(checkResponseError)
+        .then((response: Response) => response.text())
+        .then((data) => {
```

Comment:
Please remove the `()` or specify the type, as the compiler currently thinks that the type is `any`